workspaces/marketplace/plugins/marketplace-common/src/api/MarketplaceCatalogClient.test.ts

```diff
@@ -93,13 +93,13 @@ describe('MarketplaceCatalogClient', () => {
 
     it('should return the plugins', async () => {
       const api = new MarketplaceCatalogClient(options);
-      const plugins = await api.getPlugins();
+      const plugins = await api.getPlugins({});
```

Comment:
argument is optional, isnt it? 🤔 
```suggestion
      const plugins = await api.getPlugins();
```